Helper/Paymentorder.php

```diff
@@ -176,7 +177,6 @@ public function createPaymentorderObject($consumerProfileRef = null)
         $urlData = $this->createUrlObject();
         $payeeInfo = $this->createPayeeInfoObject();
         $orderItems = $this->createOrderItemsObject($mageQuote);
-        $payer = $this->createPayerObject($mageQuote);
```

Comment:
This change doesn't make any sense to me. `payer` should be set if Checkin is enabled. Also, MSISDN should be sent if available, but it should not be required. When MSISDN is sent, it should include a country prefix.

Reply:
When a customer checks in through the check-in widget, the phone number is used to populate the phone number field in Magento. The same thing happens when a customer proceeds as guest and fills in the details manually. If we want to send in the MSISDN, then we need to implement a validator so that Swedbank Pay receives the phone number in correct format.

However, as we were told by Gillis, Swedbank Pay does not need phone number information. Therefore, the task was to remove phone number, thus payer object from the request

Reply:
It's okay removing MSISDN, but not the entire `payer` object.

Reply:
Last time I checked the documentation, the `msisdn` field was required for the `payer` object. Has the documentation been updated recently?

Reply:
No, `msisdn` nor any of the other properties beneath `payer` are required.

Reply:
All right. I can add it back then! 👍

Reply:
Please do

Reply:
`payer` object has been added back to the request